cpp/src/round/round.cu

```diff
@@ -31,6 +33,9 @@ namespace {  // anonymous
 float __device__ generic_round(float f) { return roundf(f); }
 double __device__ generic_round(double d) { return ::round(d); }
 
+float __device__ generic_round_half_even(float f) { return rintf(f); }
+double __device__ generic_round_half_even(double d) { return ::rint(d); }
```

Comment:
Why the `::` here but not on `rintf` above?

Reply:
Good point, might not be needed.

Reply:
You're right, fixed.